clang/test/CodeGen/fp-accuracy.c

```diff
@@ -396,7 +396,7 @@ void f1(float a, float b) {
 // CHECK: call double @llvm.fpbuiltin.tan.f64(double {{.*}}) #[[ATTR_HIGH]]
 // CHECK: call double @llvm.fpbuiltin.log10.f64(double {{.*}}) #[[ATTR_HIGH]]
 // CHECK: call void @llvm.fpbuiltin.sincos.f64(double {{.*}}, ptr {{.*}}, ptr {{.*}}) #[[ATTR_HIGH]]
-// CHECK: call float @tanf(float {{.*}})
+// CHECK: call float @llvm.fpbuiltin.tan.f32(float {{.*}}  #[[ATTR_HIGH]]
```

Comment:
Should have caught this earlier, but please add a top-level comment explaining what this test does.